### DISCUSSION TITLE
[Urgent] Assistance Needed

### DISCUSSION THREAD

**Discussion by AshishSolanki4172** [community/community#124210] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am writing on behalf of my friend, Pulkit Kumar Mathur, whose GitHub account (username: Pulkit1822) was suspended on May 13, 2024, without prior notification. This account is crucial for his college work and job applications, and its suspension is causing him a lot of trouble.

Despite raising Ticket ID 2781440 immediately, he has not received any response or update. It is very disappointing that no one has addressed this issue despite multiple attempts to contact support and their officials. This has been a very bad experience for him.

**Comment by ebndev** [comment]
Hi @AshishSolanki4172,

This community is managed by GitHub employees, but it is not a support forum. The user has done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
